newrelic/newrelic-python-agent
# Fix local scoped package reporting

# Overview
* Fixes a bug where locally scoped packages where not being reported on connect.
* Fixes a bug where importlib_metadata version was not being captured correctly.

# Testing
* Checkout main.
* `pip install -e <newrelic>`
* `pip install --user <modules>`
* Run a simple web app and via the plugins in the UI under the environment tab. The packages installed under user scope should be missing.
* Checkout this branch and re-run the test, now the packages installed under user scope should be listed in the UI.
